app/jobs/mailchimp/email_updater_job.rb

```diff
@@ -0,0 +1,11 @@
+module Mailchimp
+  class EmailUpdaterJob < ApplicationJob
+    def perform(user:, original_email:)
+      contact = Mailchimp::Contact.from_user(user)
+      Mailchimp::AudienceManager.update_contact(contact, original_email)
+    rescue => e
+      Rollbar.log(e, job: :audience_updater)
```

Comment:
is this necessary as Log.exception calls rollbar too?

Reply:
Should have paid closer attention to what `Log.exception` was doing. Removed duplicates.